test/DomainRegistry.test.js

```diff
@@ -0,0 +1,87 @@
+const { expect } = require("chai");
+
+describe("DomainRegistry", function () {
+  let domainRegistry;
+  let owner;
+
+  beforeEach(async function () {
+    const hre = require("hardhat");
+    const DomainRegistry = await hre.ethers.getContractFactory("DomainRegistry");
+    domainRegistry = await DomainRegistry.deploy();
+    [owner] = await hre.ethers.getSigners();
+  });
+
+  it("Should register a new .com domain", async function () {
+    const domainForThisTest = "com";
+    const tx = await domainRegistry.connect(owner).registerDomain(domainForThisTest, {
```

Comment:
Why do you `connect(owner)` each time? It is by default the first signer anyway and you just increase the code size.
-1 points